index.test.ts

```diff
@@ -375,4 +375,35 @@ describe('Sort JSON', () => {
 
     expect(output).toMatchSnapshot();
   });
+
+  it('should sort JSON objects within an array', () => {
```

Comment:
Could we add a second "validate" test case as well?

So far with each fixture, I've tested the case where it _requires_ sorting, and the case where it's _already_ sorted correctly. Just to ensure that it won't ever change a valid fixture.

Reply:
Just made a commit adding this new case.